tests/traits/test_with_timestamp.py

```diff
@@ -98,22 +101,22 @@ def test_get_timetamp_or_duration_day_of_month(wait):
 
 
 @pytest.mark.usefixtures("wait")
+@freeze_time("2017-02-03 11:01:02")  # 11:01 UTC is actually 12:01 Paris time that day.
 def test_get_timetamp_or_duration_day_of_month_at(wait):
     wait2 = copy.deepcopy(wait)
-    now = datetime.datetime.now()
-    hour = now.hour
-    wait.day_of_month(now.day).at("{}:{}:{}".format(hour + 1, now.minute, now.second))
+    now = datetime.datetime.now(pytz.timezone('Europe/Paris'))
+    wait.day_of_month(3).at("13:01:02")
     assert wait.get_timetamp_or_duration()[0] - int(now.timestamp()) == 60 * 60
-    wait2.day_of_month(now.day).at("{}:{}:{}".format(hour - 1, now.minute, now.second))
-    assert 28 * 24 * 60 * 60 <= wait2.get_timetamp_or_duration()[0] - int(now.timestamp()) <= 31 * 24 * 60 * 60
```

Comment:
If February has 28 days, then we might wait for 28 days minus an hour, so that assertion was false. With a fixed time, it's now an equality assertion.